db/migrations/20210308194135-update_posts_table.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+ALTER TABLE posts
+  ADD COLUMN body_hash text NOT NULL default '';
```

Comment:
Wasn't sure if should rename the column instead... then add a new `body` column...